client/components/tooltip/tooltip-base.tsx

```diff
@@ -153,6 +190,10 @@ const TooltipBase = ( {
 				return;
 			}
 
+			if ( ! ( wrappedElement instanceof HTMLElement ) ) {
+				return;
+			}
```

Comment:
Using `instanceof HTMLElement` to ensure `wrappedElement.getBoundingClientRect()` exists before calling.